NugetLightRequest.cs

```diff
@@ -58,9 +58,6 @@ public abstract class NuGetRequest : Request {
         private HttpClient _httpClient;
         private HttpClient _httpClientWithoutAcceptHeader;
         private bool? _isCalledFromPowerShellGet;
-        private static readonly string _dollarPSHome = "$PSHome";
-        private static bool _isWindow;
-        private static bool _psHomePathInitialized;
```

Comment:
It doesn't seem to be used anywhere. Running *-Package cmdlets with newly built NuGetProvider seems to work fine.